src/style.css

```diff
@@ -1 +1,120 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form__big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form__small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-form__label::after {
+  position: absolute;
+  content: '';
+  display: block;
+}
+
+.search-form__label__big::after {
+  background-image: url('images/Search.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
```

Comment:
it is common properties for both big and small labels. So move it under common selector